src/applications/edu-benefits/tests/components/SchoolSelectField.unit.spec.jsx

```diff
@@ -291,46 +291,90 @@ describe('<SchoolSelectField>', () => {
       state: 'testState',
       zip: '12345',
       country: 'USA'
-    }];
-    const tree = mount(<SchoolSelectField
-      formData={{}}
-      formContext={{}}
-      currentPageNumber={1}
-      institutionQuery="test"
-      institutions={institutions}
-      onChange={onChange}
-      manualSchoolEntryChecked={false}
-      pagesCount={2}
-      searchInputValue="test"
-      searchResultsCount={1}
-      selectInstitution={selectInstitution}
-      showInstitutions
-      showInstitutionsLoading={false}
-      showPagination
-      showPaginationLoading={false}/>
-    );
-
-    tree.find('#page-1-0').first().simulate('change');
-    expect(onChange.firstCall.args[0]).to.eql({
-      address: {
-        city: 'testcity',
-        country: 'United States',
-        postalCode: '12345',
-        state: 'testState',
-        street: 'testAddress1',
-        street2: 'testAddress2'
-      },
-      name: 'testName',
-      'view:facilityCode': 'test'
-    });
-    expect(selectInstitution.firstCall.args[0]).to.eql({
+    };
+    const internationalInstitution = {
       address1: 'testAddress1',
       address2: 'testAddress2',
       address3: 'testAddress3',
       city: 'testcity',
       facilityCode: 'test',
       name: 'testName',
-      state: 'testState'
+      state: 'testState',
+      zip: '12345',
```

Comment:
Do international institution addresses use the zip and state fields?

Reply:
I'm not sure and given that the international address formats we get back from the search tool are a little weird I don't think we can count on them always missing them. @rianfowler likely knows more about the different address formats we get from the search tool.

Reply:
is nothing sacred?